app/main.py

```diff
@@ -1,8 +1,46 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str
+                 ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        return sum(self.wash_single_car(car) for car in cars)
```

Comment:
We can wash the car only if `clean_mark < clean_power`. You must filter cars in this method to avoid unnecessary calls of `wash_single_car`.

Reply:
In `wash_single_car` I filter cars, so I can use the `wash_single_car` method to simplify and shorten the code, I should rewrite the code, or let it be?

Reply:
Yes, rewrite it, please

Reply:
I had an error "AssertionError: Expected '`wash_single_car`' to have been called inside '`serve_cars`' method", so even pytest says, that we need to use `wash_single_car` in `serve_cars`.